src/chainparams.cpp

```diff
@@ -124,7 +124,10 @@ class CMainParams : public CChainParams {
         assert(consensus.hashGenesisBlock == uint256S("0x000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f"));
         assert(genesis.hashMerkleRoot == uint256S("0x4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b"));
 
-        // Note that of those with the service bits flag, most only support a subset of possible options
+        // Note that of those with the service bits flag, most only support a subset of possible options.
```

Comment:
You still mention the "service bits flag".

Reply:
Ah, I was thinking of the xXXXX prefix as the "service bits flag"...

Reply:
Ah, yes, that's also a possible interpretation. Thanks for clarifying the text.